portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,21 +34,22 @@
 /** Servlet that handles comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  public static final String PARAMETER = "number-of-comments";
```

Comment:
Change the name since 'PARAMETER' is very vague.

Reply:
Done.